source/irfs/full_enclosure/bkg/index.rst

```diff
@@ -58,7 +58,7 @@ Required columns:
 * ``ENERG_LO``, ``ENERG_HI`` -- ndim: 1, unit: TeV
     * Reconstructed energy axis
 * ``DETX_LO``, ``DETX_HI``, ``DETY_LO``, ``DETY_HI`` -- ndim: 1, unit: deg
-    * Field of view coordinates binning, see :ref:`coords-fov`
+    * Field of view coordinates binning (see :ref:`coords-fov`)
```

Comment:
These are of course also somewhat ambiguous, it's not clear along which system these are aligned. Add a note that this should be specified clearly?

Reply:
I have written long comments / notes in the general FOV section. Does it really help to partly re-hash those comments here?
If you think so, would you be willing to add a commit to this branch with any text changes you like?

Reply:
Agreed, this is probably not necessary. Ignore my comment.